content/ecosystem/sdks/sdks.toml

```
@@ -0,0 +1,20 @@
+[matrix-rust-sdk]
+language = "Rust"
+licence = "Apache-2.0"
+repository = "https://github.com/matrix-org/matrix-rust-sdk"
+purpose = ["client", "bot"]
```

```suggestion
purpose = ["client", "bot", "bridge"]
```

unless you want to list the `matrix-sdk-appservice` Rust crate separately? (it's in the same repository)